File: pedro-api/src/main/java/br/com/cpcx/web2/pedro/domain/entity/Produto.java
package br.com.cpcx.web2.pedro.domain.entity;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;
import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "TB_PRODUTO")
public class Produto {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "PRO_ID")
    private Long id;

    @Column(name = "PRO_DESCRICAO")
    private String descricao;

    @Column(name = "PRO_QUANTIDADE_ESTIQUE")
    private Integer quantidadeEstoque;

    @Column(name = "PRO_IDADE_PERMITIDA")
    private Integer idadePermitida;

    @Column(name = "PRO_PRECO_COMPRA")
    private BigDecimal precoCompra;

    @Column(name = "PRO_PRECO_VENDA")
    private BigDecimal precoVenda;

}

File: pedro-app/src/models/produto/Produto.js
export default Produto;

class Produto{
    constructor(id, descricao, quantidadeEstoque, idadePermitida, precoCompra, precoVenda) {
        this._id = id;
        this._descricao = descricao;
        this._quantidadeEstoque = quantidadeEstoque;
        this._idadePermitida = idadePermitida;
        this._precoCompra = precoCompra;
        this._precoVenda = precoVenda;

    }

        get id(){
            return this._id;
        }
        get descricao(){
            return this._descricao;
        }
        get quantidadeEstoque(){
            return this._quantidadeEstoque;
        }
        get idadePermitida(){
            return this._idadePermitida;
        }
        get precoCompra(){
            return this._precoCompra;
        }
        get precoVenda(){
            return this._precoVenda;
        }

}
